samples/excel/82-document/get-file-in-slices-async.yaml

```diff
@@ -0,0 +1,201 @@
+id: excel-82-document-get-file-in-slices-async-
+name: Get file in slices and recombine
+description: Get current file in slices and recombine the slices afterwards.
```

Comment:
Suggest that you change the description to:   Get the base64-encoded string that represents the current document.

(The info about getting slices, and recombining slices, is implementation details that you can mention in the snippet itself...it's not needed in the description.)

Reply:
Thanks, done.